2020-06-30 01:00:10.194976  script ran
2020-06-30 01:05:10.860752  script ran
2020-06-30 01:10:11.769354  script ran
2020-06-30 01:15:11.706452  script ran
2020-06-30 01:20:11.953630  script ran
2020-06-30 01:25:11.691665  script ran
2020-06-30 01:30:10.587645  script ran
2020-06-30 01:35:11.614163  script ran
2020-06-30 01:40:11.637675  script ran
2020-06-30 01:45:11.570813  script ran
2020-06-30 01:50:11.682922  script ran
2020-06-30 01:55:11.781288  script ran
2020-06-30 02:00:12.185611  script ran
2020-06-30 02:05:10.787130  script ran
2020-06-30 02:10:12.105125  script ran
2020-06-30 02:15:10.412266  script ran
2020-06-30 02:20:11.521208  script ran
2020-06-30 02:25:11.811927  script ran
2020-06-30 02:30:11.993859  script ran
2020-06-30 02:35:11.295416  script ran
2020-06-30 02:40:11.834165  script ran
2020-06-30 02:45:12.079410  script ran
2020-06-30 02:50:10.972208  script ran
2020-06-30 02:55:10.596936  script ran
2020-06-30 03:00:11.602882  script ran
2020-06-30 03:05:11.629524  script ran
2020-06-30 03:10:11.713044  script ran
2020-06-30 03:15:11.841088  script ran
2020-06-30 03:20:12.241061  script ran
2020-06-30 03:25:11.236961  script ran
2020-06-30 03:30:11.415207  script ran
2020-06-30 03:35:11.079302  script ran
2020-06-30 03:40:11.219805  script ran
2020-06-30 03:45:11.885780  script ran
2020-06-30 03:50:11.713260  script ran
2020-06-30 03:55:11.881338  script ran
2020-06-30 04:00:11.000724  script ran
2020-06-30 04:05:26.424797  script ran
2020-06-30 04:10:11.937370  script ran
Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 297.334
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 271.882
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 277.166
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

2020-06-30 04:55:11.093845  script ran
2020-06-30 05:00:10.626836  script ran
Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 279.681
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 295.905
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 287.811
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 33, in get_html_from_site
    liquidity_providers = driver.find_element_by_xpath(xpath_lp)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[text()='Liquidity providers']"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 298.081
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 298.881
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 298.074
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 296.091
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 294.046
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 30, in get_html_from_site
    iframe = driver.find_element_by_xpath(xpath_iframe_lp)  # switch to it
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="main-center-col"]/div/p[12]/iframe"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 33, in get_html_from_site
    liquidity_providers = driver.find_element_by_xpath(xpath_lp)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[text()='Liquidity providers']"}
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 285.676
  (Session info: headless chrome=83.0.4103.97)

Traceback (most recent call last):
  File "/home/ubuntu/trends-script/scraper.py", line 176, in <module>
    main()
  File "/home/ubuntu/trends-script/scraper.py", line 169, in main
    htmlstring = get_html_from_site()
  File "/home/ubuntu/trends-script/scraper.py", line 28, in get_html_from_site
    driver.get(start_url)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: -0.001
  (Session info: headless chrome=83.0.4103.97)

